### PR TITLE
docs: Website Revamp

### PR DESCRIPTION
**Pull request by sumeshir26**

### CODE REVIEW

**Review by not-nef — APPROVED** [review]

**Comment by im-coder-lg** [comment]
I'll inspect now before revision

File: docs/src/pages/HomepageFeatures.js [hunk 1]
```diff
@@ -9,20 +9,20 @@ const FeatureList = [
...
       <>
-        Uses{" "}
+        Supports{" "}
         <Link to="https://github.com/rdbende/Sun-Valley-TTk-Theme">
           {" "}
-          @rdbende's Sun Valley TTk Theme
+          Fluent Design Themeing
         </Link>
-        . Feels modern!
+        and Mica. Feels modern!
```
Comment (rdbende) [inline docs/src/pages/HomepageFeatures.js:17]: Nit:
missing space before 'and'
Themeing -> Theming

Comment (im-coder-lg) [reply docs/src/pages/HomepageFeatures.js:17]: I'll try fixing these now.

**Review by im-coder-lg — COMMENTED** [review]
Did what I could. Inspect it and if it has errors,report to me, I'll fix it.

**Comment by im-coder-lg** [comment]
It's due to the frontmatter not formatted I guess. Will fix it in a few mins, plz wait.

**Comment by im-coder-lg** [comment]
Look at the frontmatter:
https://github.com/Futura-Py/TimerX/blob/website-revamp/docs/docs/compile.md
It shouldn't have the Futura-Py part there, remove that so that there are the three dashes only.

**Comment by im-coder-lg** [comment]
> Well, I kind of liked the old text in the Theming section, wy did you change it? And aso, i thibk that mentioning python is enough no need for the tkinter.ttk part

I couldn't see your changes - god I hope it's fixed. After my exams there are holidays so I could try helping that time. Thorough with tomorrow's so just checking on here.

**Comment by im-coder-lg** [comment]
Alright, I'm back. Got some perms to code, so revamping the website!!!

**Comment by im-coder-lg** [comment]
I have an idea. WHy not port [this](https://github.com/kimlimjustin/xplorer/blob/master/docs/src/pages/support.jsx)(yeah its another repo) here? I mean, it shows contributor count, or we could use the typewriter effect, right? Or maybe just do something on our own.

**Comment by im-coder-lg** [comment]
I am testing some things like the new CSS I took over from [here](https://github.com/kimlimjustin/xplorer/blob/master/docs/src/css/custom.css) but anything might change!

**Review by not-nef — APPROVED** [review]

**Comment by im-coder-lg** [comment]
Not done yet.

**Comment by sumeshir26** [comment]
@im-coder-lg are you done?

**Comment by im-coder-lg** [comment]
Unfortunately, no. I have a bunch of local commits, will push when done with all the homework.

**Comment by im-coder-lg** [comment]
I think it's good for merge, I'll add my changes later. I want to add a gradient bg and a better TimerX icon for the sidebar.

**Pull request merged**